src/ServerConfig.chpl

```diff
@@ -282,7 +283,8 @@ module ServerConfig
             regexMaxCaptures = regexMaxCaptures,
             byteorder = try! getByteorder(),
             autoShutdown = autoShutdown,
-            serverInfoNoSplash = serverInfoNoSplash
+            serverInfoNoSplash = serverInfoNoSplash,
+            maxArrayDims = MaxArrayDims
```

Comment:
just a note for other reviewers, it wasn't immediately obvious where `MaxArrayDims` was originally defined but I found it as a config var on line 45. and if i'm to trust the comment about it, it should get set by one of the python scrips